src/net/client/auth/JwtBearerSession.cpp

```
@@ -0,0 +1,212 @@
+  LogFmt("{}: authenticating (user len={})", config.log_tag, login_id.size());
+
+  const boost::json::object body{
+    {config.login_id_json_field, login_id},
+    {config.login_secret_json_field, login_secret},
+  };
+  StringOutputStream post_stream;
+  Json::Serialize(post_stream, body);
+  const std::string post_data = post_stream.GetValue();
+
+  CurlSlist headers;
+  headers.Append("Content-Type: application/json");
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function declared not to throw exceptions.

(throwInNoexceptFunction)